src/test/java/se/sundsvall/emailreader/service/EmailServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package se.sundsvall.emailreader.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+class EmailServiceTest {
+
+    @InjectMocks
+    private EmailService emailService;
+
+
+    @Test
+    void testGetAllEmails() {
+
+        final var result = emailService.getAllEmails("someMunicipalityId", "someNamespace");
+
+        assertThat(result).isNotNull().isNotEmpty().hasSize(1);
```

Comment:
Not a fault here, but maybe a possibility to tune the code a bit?:

`.isNotEmpty()` is not necessary here. I.e. "is not empty" is implied when asserting that size is 1.